### DISCUSSION TITLE
How to install?

### DISCUSSION THREAD

**Discussion by DeltaBugles** [Codectory/AutoActions#3] [Q&A]
I might be daft, but how do I install the application?

**Comment by Codectory** [comment]
Just unzip the files and run the HDRProfile.exe. An installation is not
necessary.

DeltaBugles <notifications@github.com> schrieb am Di. 2. Feb. 2021 um 19:42:

**Comment by florianfelsing** [comment]
You probably downloaded the source code instead of the executable release.

Go here to download the latest x64 or x86 release, depending on your system: https://github.com/Codectory/AutoActions/releases
